### PR TITLE
Fizzbuzz

### PR DESCRIPTION
**Pull request by maksimgm**
@bengrunfeld

### CODE REVIEW

File: fizzbuzz.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+for( var i = 1; i <= 100; i++ ) {
+  var divisibleByThree = i % 3 === 0;
+  var divisibleByFive = i % 5 === 0;
```
Comment (bengrunfeld) [inline fizzbuzz.js]: you should only really have assignment or calculations going on in a statement, not both. So `var d = i % 3;` Then later `if (d === 0) {}`

File: fizzbuzz.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+  if( divisibleByThree) {
```
Comment (bengrunfeld) [inline fizzbuzz.js]: Better like this:

    if (i % 3 === 0) {
        //do something here
    }

File: fizzbuzz.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    document.write("fizz" + "<br>");
```
Comment (bengrunfeld) [inline fizzbuzz.js]: Don't use `document.write`. Use `console.log` or `getElementById` then change its text.

File: fizzbuzz.js (hunk 1 above)
Comment (maksimgm) [inline fizzbuzz.js]: Are you referring to var divisibleByThree and divisibleByFive?

Comment (bengrunfeld) [inline fizzbuzz.js]: yup

**Pull request merged**